libbeat/processors/translate_ldap_attribute/docs/translate_ldap_attribute.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[processor-translate-guid]]
+=== Translate GUID
+
+++++
+<titleabbrev>translate_ldap_attribute</titleabbrev>
+++++
+
+The `translate_ldap_attribute` processor translates an LDAP attributes between eachother.
```

Comment:
```suggestion
The `translate_ldap_attribute` processor translates LDAP attributes between entities.
```
I'm not sure if "entities" is the right word. Just a guess on my part. :-)

Reply:
It translates an attribute of an entity to another one of the same entity eg Get a Name based on an ID